### PR TITLE
Modify Ansible resources for IS 5.10.0

### LINKED ISSUE #40: JDK unzipping failure
**Description:**
Sometimes the JDK archive name is different from the extracted folder name. Current Ansible scripts expect them to be the same.

**Suggested Labels:**
Type/Bug
Priority/Normal

**Suggested Assignees:**
@chirangaalwis

### LINKED ISSUE #44: Modify configurations for Identity Server 5.10.0
**Description:**
The configurations should be modified to support Puppet resources for IS 5.10.0

### PR DESCRIPTION
**Pull request by Savidude**

### CODE REVIEW

File: dev/inventory [hunk 1]
```diff
@@ -1,13 +1,5 @@
 [is]
-is_1 ansible_host=[ip_address] ansible_user=[ssh_user]
+is_1 ansible_host=172.28.128.5 ansible_user=vagrant
```
Comment (VimukthiPerera) [inline dev/inventory]: Shall we change this to place holders?

File: README.md [hunk 2]
```diff
@@ -44,65 +41,25 @@ This repository contains the Ansible scripts for installing and configuring WSO2
...
+│           ├── carbon-home
+│           │   ├── bin
+│           │   │   └── wso2server.sh.j2
+│           │   └── repository
+│           │       ├── conf
+│           │       │   └── deployment.toml.j2
+│           │       └── deployment
+│           │           └── server
+│           │               └── eventpublishers
+│           │                   ├── IsAnalytics-Publisher-wso2event-AuthenticationData.xml.j2
```
Comment (VimukthiPerera) [inline README.md]: @Savidude We do not need these XML files as analytics is removed. Shall we remove them as well?

File: dev/inventory (hunk 1 above)
Comment (Savidude) [reply dev/inventory]: Fixed https://github.com/wso2/ansible-is/pull/45/commits/cf8983700c30d2f4374691203bd4f7d737dd1b11

File: README.md (hunk 2 above)
Comment (Savidude) [reply README.md]: Removed eventpublishers https://github.com/wso2/ansible-is/pull/45/commits/0dbe8a4f2bdb747caea0f887882b105350769c09

**Pull request merged**